### DISCUSSION TITLE
Github Copilot menu not shown in IntelliJ

### DISCUSSION THREAD

**Discussion by ivanrome** [community/community#7320] [Copilot Conversations] [answered]
Hello,

After installing the plugin in IntelliJ, logging and accept terms

![image](https://user-images.githubusercontent.com/13920663/140423069-c0278962-5959-4d06-b788-0b9643037784.png)
 the  File menu doesn't have the Github copilot entry
![image](https://user-images.githubusercontent.com/13920663/140423514-e9556c49-af5c-423c-b4bc-066500c81f93.png)

How can I activate it?

**Comment by martin-marko** [comment, marked as answer]
It's under Tools - GitHub Copilot
